EXOSIMS/Prototypes/PlanetPhysicalModel.py

```diff
@@ -37,6 +39,20 @@ def __init__(self, cachedir=None, **specs):
         # load the vprint function (same line in all prototype module constructors)
         self.vprint = vprint(specs.get('verbose', True))
         
+        #Select which Phase Function to use
+        assert isinstance(whichPlanetPhaseFunction, str), "whichPlanetPhaseFunction is not a string"

```

Comment:
Ok, but this directly contradicts the new code in BrownCompleteness (`if callable(self.PlanetPhysicalModel.whichPlanetPhaseFunction)`).  Here you are forcing the input to be a string.  If you want to go this route, then get rid of the callable logic in BrownCompleteness.  I'm ok with punting on making this an object input.  Let's just keep it as a string.  You also need to make sure that whatever changes you made in BrownCompleteness are working for GarrettCompleteness.